src/database/queries/organisationExtension.js

```diff
@@ -86,4 +86,13 @@ module.exports = class OrganizationExtensionQueries {
 			throw new Error(`Error fetching organisation extension: ${error.message}`)
 		}
 	}
+
+	static async create(data, options = {}) {
+		try {
+			const newOrgExtension = await OrganizationExtension.create(data, options)
```

Comment:
@joffinjoy why is it not consistently  camel case OrganizationExtension => organizationExtension